model/ftn/gkeModule.f90

```diff
@@ -0,0 +1,1908 @@
+!/ ------------------------------------------------------------------- /
```

Comment:
Is there a reason this is not a *.ftn file and is not named in the typical way such as w3gkemd.ftn?

Reply:
No particularly reasons. I just followed the way how NL2 organizes the modules. I renamed this file as w3gkemd.ftn as suggested.